docs/source/endpoints/content-rules.md

```diff
@@ -0,0 +1,169 @@
+---
+myst:
+  html_meta:
+    "description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+    "property=og:description": "Available content rules in a Plone site can be listed and queried by accessing the /@content-rules endpoint on any context. Access requires the ManageContentRules permission."
+    "property=og:title": "Content Rules"
+    "keywords": "Plone, plone.restapi, REST, API, ContentRules, Rules"
+---
+
+(content-rules)=
+
+# Content Rules
+
```

Comment:
````suggestion

A content rule will automatically perform an action when a certain event, known as a {term}`trigger`, takes place.

Content rules can perform routine tasks, including the following.

-   Move content from one folder to another when that content item is published.
-   Send email when a content item is deleted.
-   Delete content after a certain date.

```{seealso}
[Using Content Rules](https://docs.plone.org/working-with-content/managing-content/contentrules.html)
```

````

Also add terms to the Glossary.

```
content rule
    A content rule will automatically perform an action when a certain event, known as a {term}`trigger`, takes place.

trigger
    A trigger is an event in Plone that causes the execution of defined actions.
    Example triggers include object modified, user logged in, and workflow state changed.
```